doc/source/myrocks/limitations.rst

```diff
@@ -6,30 +6,20 @@ MyRocks Limitations
 
 The MyRocks storage engine lacks the following features compared to InnoDB:
 
-* `Online DDL <https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl.html>`_
-
-* `ALTER TABLE ... EXCHANGE PARTITION
-  <https://dev.mysql.com/doc/refman/8.0/en/partitioning-management-exchange.html>`_
-
-* `SAVEPOINT <https://dev.mysql.com/doc/refman/8.0/en/savepoint.html>`_
-
-* `Transportable tablespace <https://dev.mysql.com/doc/refman/8.0/en/innodb-transportable-tablespace-examples.html>`_
-
-* `Foreign keys <https://dev.mysql.com/doc/refman/8.0/en/create-table-foreign-keys.html>`_
-
-* `Spatial indexes <https://dev.mysql.com/doc/refman/8.0/en/using-spatial-indexes.html>`_
-
-* `Fulltext indexes <https://dev.mysql.com/doc/refman/8.0/en/innodb-fulltext-index.html>`_
-
-* `Gap locks <https://dev.mysql.com/doc/refman/8.0/en/innodb-locking.html#innodb-gap-locks>`_
-
-* `Generated Columns
-  <https://dev.mysql.com/doc/refman/8.0/en/create-table-generated-columns.html>`_
-
-* `Group Replication <https://dev.mysql.com/doc/refman/8.0/en/group-replication.html>`_
-
-* `Partial Update of LOB in InnoDB <https://mysqlserverteam.com/mysql-8-0-optimizing-small-partial-update-of-lob-in-innodb/>`_
-
+   * `Online DDL <https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl.html>`_ is not supported due to lack of atomic DDL support. 
+       - There is no ``ALTER TABLE ... ALGORITHM=INSTANT`` functionality
+       - Partition management operations only supports ``COPY`` algorithms    which rebuild the partition table and moves data based on the new ``PARTITION ... VALUE`` definition. In the case of ``DROP PARTITION``, the data which cannot be moved to another partition is deleted.
```

Comment:
s/supports/support
extra spaces between 'algorithms    which'

Reply:
Removed spaces
rewrote the sentences.